bin/steps/sqlite3

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set +e
```

Comment:
This steps file is sourced, so the `set +e` here will affect subsequent steps too. This may be fine, but mentioning it just in case.